### PR TITLE
Testing utils

### PR DESCRIPTION
**Pull request by mictab**
* Adds a test requirement
* Adds factories (so far for Fair, Company and Event)

Later the current (and future tests) would make use of the factories.

### CODE REVIEW

File: events/factories.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import factory
+from django.utils import timezone
+
+from fair.factories import FairFactory
+from .models import Event
+
```
Comment (Tiseno) [inline events/factories.py]: What is the logic of structuring the imports like this and/or do we have a preferred way of structuring imports?

File: fair/models.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from django.db import models
 from datetime import date
 
+
```
Comment (Tiseno) [inline fair/models.py]: What?

Comment (mictab) [inline fair/models.py]: PyCharm doing it's PEP8 thing ;)

Comment (Tiseno) [inline fair/models.py]: rip pycharm

File: events/factories.py (hunk 1 above)
Comment (mictab) [inline events/factories.py]: Basically, according to PEP8:

1. Standard library imports
2. Third party imports
3. Local application imports

With a blank line between each group.

**Comment by mictab** [comment]
Rebasing against master.

**Comment by mictab** [comment]
Rebasing against master.

**Comment by mictab** [comment]
Rebased against master.

**Pull request merged**